snap/snapcraft.yaml

```diff
@@ -0,0 +1,26 @@
+name: ktlint
+version: git
+summary: An anti-bikeshedding Kotlin linter with built-in formatter
+description: |
+   Kotlin linter in spirit of feross/standard (JavaScript) and gofmt (Go).
+
+confinement: devmode
+
+apps:
+  ktlint:
+    command: java -jar $SNAP/ktlint-0.0.0-SNAPSHOT-shaded.jar
+    plugs: [home]
+    environment:
+      PATH: $PATH:$SNAP/usr/lib/jvm/java-8-openjdk-amd64/bin
+
+parts:
+  ktlint:
+    source: .
+    plugin: nil
+    build-packages: [openjdk-8-jdk]
+    stage-packages: [openjdk-8-jre]
+    build: |
+      ./mvnw
+      ./mvnw -Pcapsule clean package -Dmaven.test.skip=true
+    install:
+      mv ktlint/target/ktlint-0.0.0-SNAPSHOT-shaded.jar $SNAPCRAFT_PART_INSTALL
```

Comment:
[ktlint/target/ktlint](https://github.com/shyiko/ktlint/releases/tag/0.15.0) (executable jar that [doesn't require](https://github.com/shyiko/ktlint#usage) `java -jar ...` to be run) should be used instead (produced by `./mvnw -Pcapsule clean package -Dmaven.test.skip=true`).

Reply:
Awesome! updated.